5.5/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:centos7
+
+ADD mysql /opt/openshift/
+ENV PATH /opt/openshift/bin:$PATH
+
+WORKDIR /opt/openshift
+RUN create_user.sh
```

Comment:
just embed the commands directly here, i'm not seeing much value in doing it this way, plus you end up with the script in your image at the end.